orion-server/src/main/java/com/pinterest/orion/core/actions/aws/EC2Helper.java

```diff
@@ -0,0 +1,49 @@
+package com.pinterest.orion.core.actions.aws;
+
+public abstract class EC2Helper {
+
+    /**
+     * Get the number of running brokers with the given prefix
+     * @param prefix the prefix of the broker name. ex: "memq-cluster1-"
+     * @return the number of running brokers
+     */
+    public abstract int getRunningBrokerCount(String prefix);
+
+    /**
+     * Get the instance id using the full host name
+     * @param fullHostName the full host name
+     * @param region the region of the host
+     * @return the instance id
+     */
+    public abstract String getHostIdUsingHostName(String fullHostName, String region);
```

Comment:
Sorry for bringing this up again. The Java doc says the function gets `instance id` but the name has `host id`. If host id it the same as EC2 instance id, I suggest using the term `instance id` as it's more commonly used and not confusing.

Reply:
Make sense. Changed